tests/acceptance_tests/optimize_test.py

```diff
@@ -8,158 +9,164 @@
 TARGET_OPERATOR = QubitOperator("X0 X1 Z2 Y4", 1.5)
 
 
+# To regenerate the circuit, run:
+# import zquantum.core.wip.circuits as new_circuits
```

Comment:
Let's also add `sympy` import here to make the snippet self-contained.

Reply:
fixed in 4c14120